TrafficSimulator/Assets/UI/World/WorldController.cs

```diff
@@ -80,6 +81,25 @@ private void VehiclesButtonOnClicked()
             _tabs = Tabs.Vehicles;
         }
 
+        private void EnablePanel(VisualElement panel, Button button)
```

Comment:
Isn't this exactly the same function as from `StatisticsController?` The logic should be moved to a single function, and then if the different controllers need to have an `EnablePanel` with different panels and buttons they should just call that single function with the panels and buttons as parameters so there is no code duplication

Then the same suggestions as for the previous controller can be ignored here as it should be deleted anyway and fixed in the single function